### DISCUSSION TITLE
Getting ReferenceError: regeneratorRuntime is not defined with React lib react-charts

### DISCUSSION THREAD

**Discussion by itisguptak** [babel/babel#11821] [General]
Hi,

I'm getting below error while I try to use react-charts. I'm trying to code basic charts from example, but getting this error.

VM1462 react-charts.mjs:923 Uncaught ReferenceError: regeneratorRuntime is not defined
at eval (VM1462 react-charts.mjs:923)
at Module../node_modules/react-charts/dist/react-charts.mjs (VM1329 bundle.js:846)
at webpack_require (VM1329 bundle.js:20)
at eval (VM1461 PcfUsageChart.js:4)
at Module../src/components/PcfUsageChart.js (VM1329 bundle.js:1482)
at webpack_require (VM1329 bundle.js:20)
at eval (VM1460 ChartDashboard.js:4)
at Module../src/components/ChartDashboard.js (VM1329 bundle.js:1470)
at webpack_require (VM1329 bundle.js:20)
at eval (VM1378 pgAccordion.js:8)

I followed solution listed in https://andrew-flower.com/blog/Async-Await-with-React, but that didn't help.

Basically installed two packages:

@babel/plugin-transform-runtime
@babel/runtime

Modified Webpack configuration (webpack.config.js) to use the plugin with the babel-loader.

module: {
rules: [
{
// for any file with a suffix of js or jsx
test: /.jsx?$/,
// ignore transpiling JavaScript from node_modules as it should be that state
exclude: /node_modules/,
// use the babel-loader for transpiling JavaScript to a suitable format
loader: 'babel-loader',
options: {
// attach the presets to the loader (most projects use .babelrc file instead)
presets: ["@babel/preset-env", "@babel/preset-react"],
plugins: [
'@babel/transform-runtime'
]
}
}

I have babel v7

This didn't help. Can someone please help me with this.

**Comment by itisguptak** [comment]
I tried with updating .babelrc as well, but that didn't work as well. Any help will be appreciated.
